### PR TITLE
[Desynth] add Shoes

### PR DESCRIPTION
**Pull request by rdmkyran**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Makes 'Shoes' desynthable.
Removed the sub skills for all desynthesis.

## Steps to test these changes

!setskill clothcraft 8
!setskill leathercraft 34
!additem lightning_crystal
!additem shoes

Desynth the shoes.

### CODE REVIEW

File: sql/synth_recipes.sql [hunk 1]
```diff
@@ -4750,6 +4750,7 @@ INSERT INTO `synth_recipes` VALUES (75528,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,27
 INSERT INTO `synth_recipes` VALUES (75529,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,2773,2774,2774,2775,4157,0,0,2779,2779,2779,2779,1,1,1,1,'Enfeeb. Kit Poison');
 INSERT INTO `synth_recipes` VALUES (75530,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,2773,2774,2774,2775,4163,0,0,2780,2780,2780,2780,1,1,1,1,'Enfeeb. Kit Blind');
 INSERT INTO `synth_recipes` VALUES (75531,0,0,0,0,0,0,0,0,10,0,4099,4241,2773,2773,2774,2774,2775,4161,0,0,2781,2781,2781,2781,1,1,1,1,'Enfeeb. Kit Sleep');
+INSERT INTO `synth_recipes` VALUES (75532,1,0,0,0,0,8,34,0,0,0,4100,4242,12994,0,0,0,0,0,0,0,818,818,848,848,2,3,1,2,'Shoes (desynth)');
```
Comment (TeoTwawki) [inline sql/synth_recipes.sql]: To date, no desynth has been proven to use the sub-skills. Whiel you cited no sources, I looked them up my self and no source says this. Ffxiclopedia lists only the 34 leather, bg doesn't list a desynth at all (not all synth-able items are desynth-able, but BG misses a lot of ones that are) and jp wiki's don't mention either making out only source 'clopedia.

Comment (rdmkyran) [reply sql/synth_recipes.sql]: Thank you for this information, I will fix that

Comment (rdmkyran) [reply sql/synth_recipes.sql]: IDs 19941, 33530, 39933, 49956, 49968, 50502, 75517, 75518 are desynths with a sub too.
Want me to fix them at the same time ?

Comment (TeoTwawki) [reply sql/synth_recipes.sql]: yes

Comment (rdmkyran) [reply sql/synth_recipes.sql]: done

**Review by WinterSolstice8 — APPROVED** [review]
seems to check out based on the data we have

**Pull request merged**